projects/components/src/list-view/list-view.component.scss

```diff
@@ -1,46 +1,59 @@
+@import 'font';
 @import 'color-palette';
 
+$key-width: 40%;
+$value-width: 60%;
+$margin-left: 12px;
+
+@mixin grid-view {
+  width: 100%;
+  padding: 8px 0;
+  display: grid;
+  grid-template-columns: $key-width $value-width;
+  align-content: center;
+}
+
 .list-view {
   height: 100%;
   width: 100%;
   overflow: auto;
   display: flex;
   flex-direction: column;
 
-  .data-row {
-    flex: 0 0 auto;
-    min-height: 40px;
-    width: 100%;
-    word-break: break-word;
-    padding: 8px 0;
+  & > * {
```

Comment:
list-view could have a huge number of children. In such cases, this does us more harm than good. We should use a class name here instead of the wildcard.